### PR TITLE
Release v1.0.5

### PR DESCRIPTION
**Pull request by alinmigea**
@plentymarkets/team-multichannel-koboldmaki

### CODE REVIEW

File: meta/documents/changelog_de.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+## v1.0.5 (2017-06-08)
+
+### Geändert
+- Das Plugin Elastic Export ist nun Voraussetzung zur Nutzung des Pluginformats AwinCOM.
+
+### Behoben
+- Es wurde ein Fehler behoben, der dazu geführt hat, dass bei dem Barcode die Marktplatzverfügbarkeit ignoriert wurde.
+- Es wurde ein Fehler behoben, bei dem der Bestand nicht korrekt ausgewertet wurde.
+- Es wurde ein Fehler behoben, bei dem Varianten nicht in der richtigen Reihenfolge gelistet wurden.
+- Es wurde ein Fehler behoben, der dazu geführt hat, dass das Exportformat Texte in der falschen Sprache exportierte.
```
Comment (mariusbippig) [inline meta/documents/changelog_de.md:12]: Sehr gut =)

File: resources/lang/de/log.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+variationNotPartOfExportStock       = "Variante ohne Bestand."
+variationNotPartOfExportPrice       = "Variatie Preis ist negativ."
```
Comment (mariusbippig) [inline resources/lang/de/log.properties]: "Variantenpreis negativ."

File: src/Generator/AwinCOM.php [hunk 3]
```diff
@@ -44,126 +64,274 @@ public function __construct(ArrayHelper $arrayHelper)
...
+     * Get the manufacturer name.
+     *
+     * @param $variation
+     * @return string
+     */
+    private function getManufacturer($variation):string
+    {
+        if(isset($this->manufacturerCache) && array_key_exists($variation['data']['item']['id'], $this->manufacturerCache))
+        {
+            return $this->manufacturerCache[$variation['data']['item']['id']];
```
Comment (mariusbippig) [inline src/Generator/AwinCOM.php]: Maybe its better if we select the manufacturer one time and keep it in cache for other items. If its possible, we could separate the manufacturer by manufacturer id, if we have that id.

File: src/ResultField/AwinCOM.php [hunk 4]
```diff
@@ -40,11 +44,11 @@ public function generateResultFields(array $formatSettings = []):array
     {
         $settings = $this->arrayHelper->buildMapFromObjectList($formatSettings, 'key', 'value');
 
-        $reference = $settings->get('referrerId') ? $settings->get('referrerId') : -1;
+        $reference = $settings->get('referrerId') ? $settings->get('referrerId') : self::ALL_MARKET_REFERENCE;
```
Comment (mariusbippig) [inline src/ResultField/AwinCOM.php:47]: 👍

**Review by mariusbippig — CHANGES_REQUESTED** [review]

**Review by hschwab — APPROVED** [review]

**Review by mariusbippig — APPROVED** [review]

**Pull request merged**